admin_ui/src/components/EditRowForm.vue

```diff
@@ -111,18 +113,26 @@ export default Vue.extend({
                 this.$store.commit("updateApiResponseMessage", message)
             } catch (error) {
                 const data = error.response.data
+                const databaseError = data["db_error"]
+                const validationError = data["detail"]
+                let errorsArray = []
+
+                if (databaseError) {
+                    errorsArray.push(databaseError)
+                } else {
+                    validationError.forEach((item: any) => {
+                        errorsArray.push(`Field ${item.loc[1]} - ${item.msg}`)
+                    })
+                }
```

Comment:
As we're repeating the logic from above, we should try and move it to shared function called something like `parseErrorResponse`.